### ISSUE TITLE
Bump version and publish on npm

### ISSUE THREAD

**Issue by safareli** [fantasyland/daggy#10]
We should change version to `1.0.0` and publish it on npm.

**Comment by DylanRJohnston** [comment]
Is this going to happen at some point?

**Comment by safareli** [comment]
@SimonRichardson can you publish it?

**Comment by davidchambers** [comment]
I'll take care of it. :)

**Comment by davidchambers** [comment]
I've created the [fantasyland][1] organization on npm to make managing permissions much easier.

@puffnfresh, could you run the following command?

```console
$ npm access grant read-write fantasyland:owners daggy
```


[1]: https://www.npmjs.com/org/fantasyland

**Comment by safareli** [comment]
@davidchambers, Brian has already executed that command, so I think this issue could be resolved now

**Comment by davidchambers** [comment]
I've released [`v1.0.0`][1]. You're welcome to create a GitHub release for the tag, @safareli.


[1]: https://github.com/fantasyland/daggy/tree/v1.0.0

**Comment by davidchambers** [comment]
```console
$ for name in $(curl -s https://api.github.com/orgs/fantasyland/repos | jq -r 'map(.name) | sort | .[]') ; do \
>   printf '\n%s\n' $name ; \
>   (npm owner ls $name 2>/dev/null | cut -d ' ' -f 1 | sort | xargs printf -- '- %s\n') ; \
> done

daggy
- davidchambers
- drboolean
- jaforbes
- joneshf
- kennknowles
- mudge
- puffnfresh
- raimohanska
- raynos
- robotlolita
- safareli
- simonrichardson
- trevorbasinger

fantasy-birds
- schtoeffel

fantasy-check
- joneshf
- puffnfresh
- stickupkid

fantasy-cofrees
- puffnfresh

fantasy-combinators
- puffnfresh

fantasy-consts

fantasy-coproducts

fantasy-eithers
- joneshf
- puffnfresh
- trevorbasinger

fantasy-equality
- stickupkid

fantasy-examples

fantasy-frees
- stickupkid

fantasy-helpers
- stickupkid

fantasy-identities
- puffnfresh

fantasy-io
- puffnfresh

fantasy-land
- davidchambers
- joneshf
- kennknowles
- laiff
- puffnfresh
- stickupkid

fantasy-lenses
- joneshf
- puffnfresh

fantasy-maybes

fantasy-monoids

fantasy-profunctors

fantasy-promises
- puffnfresh
- trevorbasinger

fantasy-readers
- joneshf
- puffnfresh
- trevorbasinger

fantasy-sorcery
- joneshf
- puffnfresh

fantasy-states
- joneshf
- puffnfresh

fantasy-stores
- puffnfresh

fantasy-these

fantasy-tuples
- joneshf
- puffnfresh
- stickupkid
- trevorbasinger

fantasy-validations
- puffnfresh
- trevorbasinger

fantasy-writers
- puffnfresh
- srijs

function-prototype-map

sweet-fantasies
```

@stoeffel, please run:

```console
$ npm access grant read-write fantasyland:owners fantasy-birds
```

@SimonRichardson, please run:

```console
$ npm access grant read-write fantasyland:owners fantasy-equality
$ npm access grant read-write fantasyland:owners fantasy-frees
$ npm access grant read-write fantasyland:owners fantasy-helpers
```

@puffnfresh, please grant `fantasyland:owners` access to the other npm packages. :)
